src/scripts/cfo_supervisor.sh

```diff
@@ -11,10 +11,10 @@ git --version
 while true
 do
     rm -rf ./tigerbeetle
-    git clone https://github.com/tigerbeetle/tigerbeetle tigerbeetle
+    git clone https://github.com/tigerbeetle/tigerbeetle tigerbeetle || true
```

Comment:
If `git clone`  fails, won't `cd tigerbeetle` also fail?

Reply:
Good point. Maybe we should only `set -u`? If we're adding `|| true` everywhere it's the same!

Reply:
Is it possible that `cd tigerbeetle` would fail, and then `cd ../` would succeed, though?

Reply:
(and replace the relative `cd` with absolute paths)

We could also keep it top level, then `set +e` at the start of the loop and `set -e` at the end (I think :D).

And perhaps a `sleep 1`...

Reply:
Although. There's no reason we can't `scp` a zig binary to the machine :upside_down_face:

Reply:
Ah, good point, that cd was a bad idea from the start, no need to mutate shell's global state in the control loop

Reply:
Added a sleep 1 for a good measure --- indeed seems nice to protect from runaway CPU loops.

I'd rather keep `-e` and stick with `|| true` though, I do like ignoring only _specific_ errors, even if it's most of them

Reply:
>Although. There's no reason we can't scp a zig binary to the machine 🙃

That's a very subversive though, I'll entertain it for a while! One benefit of the script is that it's easier to vi it in place just to try stuff out...

But, yeah, if at any point we move to a proper infra-as-code here, I agree that the boostrap seed should be a Zig binary.

Reply:
swapped `|| sleep 10` for  `|| true` which does a larger timeout, but only when we need one!

Reply:
Ok, so it seems that `( loop body ) || sleep` is actually the good shape for an sh control loop. 

I _think_ I am done with forecpushes here, but let me `sleep 600` to make sure :rofl: